src/sonic-host-services/setup.py

```diff
@@ -10,17 +10,24 @@
     url = 'https://github.com/Azure/sonic-buildimage',
     maintainer = 'Joe LeVeque',
     maintainer_email = 'jolevequ@microsoft.com',
+    packages = [
+        'host_modules'
+    ],
     scripts = [
         'scripts/caclmgrd',
         'scripts/hostcfgd',
         'scripts/procdockerstatsd',
         'scripts/determine-reboot-cause',
         'scripts/process-reboot-cause',
+        'scripts/sonic-host-server'
     ],
     install_requires = [
         'Jinja2>=2.10',
         'sonic-py-common',
         'swsssdk>=2.0.1',
+        'dbus-python',
+        'PyGObject',
+        'systemd-python',
```

Comment:
Please organize these alphabetically and pin versions of external dependencies to latest known working versions, if possible.

Reply:
Ordered the packages alphabetically.